messaging/infinispan-grpc-kafka/README.md

```diff
@@ -14,7 +14,7 @@ Verifies SASL authentication through Quarkus Kafka client extension.
 Endpoint `SaslKafkaEndpoint` is able to produce events
 and consume events and check topics through `AdminClient` and `KafkaConsumer`.
 
-## Quarkus Infinispan scenario 
+## Quarkus Infinispan scenario
 
 ##Infinispan server SSL/TLS
```

Comment:
As I understand Michal V recently added support to generate certs automatically, so some content in this section may be outdated

Reply:
I've rechecked the readme and cleaned it up a bit. There was a lot of references to stuff, that are no longer part of this module.